examples/pylab_examples/load_converter.py

```diff
@@ -1,20 +1,30 @@
 from __future__ import print_function
-from matplotlib.dates import bytespdate2num
-#from matplotlib.mlab import load
 import numpy as np
-from pylab import figure, show
+import matplotlib.pyplot as plt
 import matplotlib.cbook as cbook
+import matplotlib.dates as mdates
+
+
+# Note: matplotlib.dates doesn't have bytespdate2num.
```

Comment:
It was added in February https://github.com/matplotlib/matplotlib/commit/0a24a56ea536698ca25572d4e089b6830ef346f4 so it wound be in any released version.